STYLE_GUIDE.md

```diff
@@ -71,18 +71,18 @@
 
     ```sh
     .
-    └── src/
-        ├── stores/
-        │   ├── program-store/
-        │   │   ├── ProgramStore.js
-        │   │   └── programs.json
-        │   └── TopicStore.js
-        ├── use-cases/
-        │   └── search-programs/
-        │       ├── SearchProgramsUseCase.js
-        │       ├── SearchProgramsController.js
-        │       └── searchProgramsRoutes.js
-        └── app.js
+    |-- package.json
+    |-- src
+    |   |-- Product
+    |   |   |-- Application
+    |   |   |   `-- SearchProductsService.ts
+    |   |   |-- Domain
+    |   |   |   `-- SearchProductsBySKUService.ts
+    |   |   `-- Infrastructure
+    |   |       `-- Stores
+    |   |           `-- ProductAPIStore.ts
+    |   `-- application.ts
+    `-- tsconfig.json
```

Comment:
* please, remove ts stuff from the example, I think an example with a plain js is more clear (that means, remove tsconfig, and change ts extensions to js)
* follow the naming recommendation for folders (kebab-case)
* Use https://tree.nathanfriend.io/ for tree generations